.github/workflows/nodejs.yml

```diff
@@ -13,8 +13,8 @@ jobs:
       - name: Use Node.js 16.x
         uses: actions/setup-node@v3
         with:
-          node-version: 16.x
+          node-version: 22.x
           cache: 'npm'

```

Comment:
Since we're using `pnpm`, might be worth looking into using the appropriate cache setting? (:
```yaml
cache: 'pnpm'
```